PrioritiesSprint3.md

```diff
@@ -0,0 +1,67 @@
+In this sprint, the Eel build is in CI and the Octopus build is in Test. Each priority has a list of subtasks that are to be completed within this sprint.
+
+Each priority has a list of subtasks that are to be completed within this sprint.  
+
+Note that ARAs will be required to implement either MCQ or Pathfinder (or both).
+
+# Develop Pathfinder Query
+## Pre-development work:
+- Design new landing page to distinguish types of interactions (templated query, pathfinder, multiquery(?)) (UI)
+- Design query selection interface - Dependencies: decisions on categories of nodes to be allowed as starting points and any other factors allowed for query selection (UI, ARAs)
+- User testing path grouping strategies (UI, ARAs)
+- Design results page path  and graph display (similar to results now) - Dependencies: understanding of the number of paths and how they will be grouped (UI, ARAs)
+- Research to identify new facets needed. e.g. facets for other categories - Dependencies: pathfinder data examples (UI, ARAs)
+- User research filter designs for new filters like metaedge type or other. (UI)
+- Provides example datasets to UI (ARAs)
+## Octopus (CI) work:
+- Evaluate performance and result quality from prototype implementations (ARAs)
+
+# MultiCURIE Query (MCQ)
+## Pre-development work:
+- Designs multi curie selection -  requires interface to add and subtract select curies (how many max?) (UI)
+- Investigate additional facets needed ( e.g. curie match facet, other specific to starting or ending node category if neccesary) (UI)
+- User research filter designs for new filters like metaedge type or other. (UI)
+- ## Eel (CI):
+- Relevant KPs implement lookup MCQ (KPs)
+- NodeNorm implements set ID method
+## Octopus (TEST):
+- TRAPI 1.5 finalized and released (ARAs, KPs, UI)
+
+# Display Full KL/AT in the UI
+## Pre-Development
+- ARAs/O&O plans for utilizing KL/AT information in their scoring approaches (ARA, O&O)
+- Define next EPC area to standardize and implement in data and UI. (EPC)
+- ## Eel (CI):
+- Last few KPs/ARAs annotate edges with KL/AT terms (See status table [here](https://docs.google.com/document/d/16agzJB0OlR8z-zU3nTivYYIo09sdayN-tu8lbrZyTXY/edit?pli=1#bookmark=id.kz0zera1i398)) (KPs, ARAs)
+## Octopus (TEST):
+- UI team implements full KL/AT tags in UI display (define paradigm to accommodate this info, which may include user friendly synonyms, new icons/badges, updated EPC modal tables, etc). (UI)
+- Determine how to display KL/AT for KPs/ARAs that have not yet implemented edge-level annotations (may fall back to infores-based annotation?) (UI)
+
+# Treats Refactor
+## Octopus (TEST):
+- UI implements "treats" refactor including provenance (UI)
+
+# Scalability
+## Eel (CI):
+- Sending Concurrent queries (MVP1&MVP2) in batch of 5,10,15,20,25,30 (Test Env)
+
+# CQS Workflows
+## Eel (CI):
+-  Test and iterate MVP1 workflows
+-  Push these workflows to TEST
+
+# Fix Failing Automated Tests
```

Comment:
This priority is not clear to me.

Reply:
Is there something particular that doesn't make sense?

Reply:
Yes, Max and I discussed it a bit (please see his comment). Essentially, I'm not sure where the numbers are coming from, and I think "Number of New Tests" should be specified.

Reply:
Do you mean specified like given a number or specified like defined what the phrase "Number of New Tests" means?

Reply:
I mean replace "Number of New Tests" with a specific number or delete it, just so folks know exactly what the target is. At present, there are over 600 test assets, so we could say something like:

For 600 test assets:
- Reduce Test Failures for ARAs to < 100
- Reduce No-Results for ARAs to < 50 
- Reduce Test Failures for CQS to < 20

I'm not sure what the "right" numbers should be, but it looks like you and Max are working that out.

Reply:
Well this is exactly the problem: We know how many tests there today.  We don't know how many there will be in a month.

Reply:
Yes, but I was suggesting that we don't worry about the future and just pick a cap. Casey's suggestion below is similar in sentiment to mine but more clearly defined.